Library/Homebrew/test/cask/audit_spec.rb

```diff
@@ -613,46 +613,46 @@ def tmp_cask(name, text)
       end
     end
 
-    describe "hosting with appcast checks" do
-      let(:message) { /please add an appcast/ }
+    describe "hosting with livecheck checks" do
+      let(:message) { /please add a livecheck/ }
 
-      context "when the download does not use hosting with an appcast" do
+      context "when the download does not use hosting with an livecheck" do
```

Comment:
```suggestion
      context "when the download does not use hosting with a livecheck" do
```
(Here and below as well.)